modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/display/context/ViewConflictsDisplayContext.java

```diff
@@ -113,6 +117,45 @@ public Map<String, Object> getReactData() {
 		}
 
 		return HashMapBuilder.<String, Object>put(
+			"ctEntriesCount",
+			_ctEntryLocalService.getCTCollectionCTEntriesCount(
+				_ctCollection.getCtCollectionId())
+		).put(
+			"ctMappingInfos",
+			() -> {
+				JSONArray ctMappingInfosJSONArray =
+					JSONFactoryUtil.createJSONArray();
+
+				List<CTMappingTableInfo> ctMappingTableInfos =
+					_ctCollectionLocalService.getCTMappingTableInfos(
+						_ctCollection.getCtCollectionId());
+
+				for (CTMappingTableInfo ctMappingTableInfo :
+						ctMappingTableInfos) {
+
+					ctMappingInfosJSONArray.put(
+						JSONUtil.put(
+							"name",
+							StringBundler.concat(
+								_ctDisplayRendererRegistry.getTypeName(
+									_themeDisplay.getLocale(),
+									_portal.getClassNameId(
+										ctMappingTableInfo.
+											getLeftModelClass())),
+								" & ",
+								_ctDisplayRendererRegistry.getTypeName(
+									_themeDisplay.getLocale(),
+									_portal.getClassNameId(
+										ctMappingTableInfo.
+											getRightModelClass())))
+						).put(
+							"tableName", ctMappingTableInfo.getTableName()
+						));
+				}
+
+				return ctMappingInfosJSONArray;
+			}
```

Comment:
This seems overkill to get the full ctMappingTableInfo since ll we need is just the count to see if the publication contains any changes. We can even combine the ctEntries and ctMappingTables count into one param to indicate whether the publication is empty.